app/uk/gov/hmrc/individualsdetailsapi/controllers/CommonController.scala

```diff
@@ -36,11 +32,11 @@ abstract class CommonController @Inject()(
 
   private[controllers] def recovery: PartialFunction[Throwable, Result] = {
     case _: MatchNotFoundException => ErrorNotFound.toHttpResponse
-    case e: AuthorisationException =>
-      ErrorUnauthorized(e.getMessage).toHttpResponse
+    case _: InsufficientEnrolments => ErrorUnauthorized("User does not have valid scopes").toHttpResponse
```

Comment:
With the addition of the audit events logic for the endpoints we no longer use this recovery method for V2. This method is also used by V1 and therefore we need to be mindful if we change it. 

Suggest pulling the audit events branch in and add this logic to the V2 recovery function leaving V1 as per current processing?